Week3/project3/project.js

```diff
@@ -0,0 +1,60 @@
+const url = "https://api.github.com/orgs/HackYourFuture/repos?per_page=100";
+  const HyfReposHttps = async ()=>{
+    const response = await axios(url);
+    return response;
+  }
+  async function check(){
+   const data= await HyfReposHttps();
+   if(data.status==200){
+    SelectElement(data)
+   }else{
+     console.log("there is an error")
+   }
+  }
+  check()
+async function SelectElement(data){
+  console.log(data)
+    let selectElement = document.getElementById('repositories');
+    data.data.forEach(rep => {
+    let option = document.createElement('option');
+    option.text = rep.name;
+    option.value = rep.id;
+    selectElement.appendChild(option);
+    SelectChang(data);
+  });
+}
+function SelectChang (data) {
+    let selectElement = document.getElementById('repositories');
+    selectElement.addEventListener('change', function(){
```

Comment:
Something to think about this code is working, but I think you can organize it more, try to write the callback in the event listener as a separate function and add it in the event listener